internal/api/impl/auth/builtin.go

```diff
@@ -0,0 +1,81 @@
+package auth
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/perses/perses/internal/api/interface/v1/user"
+	"github.com/perses/perses/internal/api/shared"
+	"github.com/perses/perses/internal/api/shared/crypto"
+	databaseModel "github.com/perses/perses/internal/api/shared/database/model"
+	"github.com/perses/perses/pkg/model/api"
+	"github.com/sirupsen/logrus"
+)
+
+type builtinEndpoint struct {
+	dao user.DAO
+	jwt crypto.JWT
+}
+
+func NewBuiltinEndpoint(dao user.DAO, jwt crypto.JWT) Endpoint {
```

Comment:
Based on what is implemented for the moment, there isno need to expose this function outside of the package